Kzl 241 document controller (#86)
kuzzleio/documentation-V2: src/sdk-reference/cpp/1/document/mDelete/index.md

---
layout: sdk.html.hbs
algolia: true
title: mDelete
description:
order: 200
---

# mDelete

> Deletes documents in the persistent data storage.

Reviewer: ```suggestion
Deletes multiple documents.
```

> Returns a [partial error]({{ site_base_path }}api-documentation/errors/#partialerror) (with status 206) if one or more document can not be deleted.

Reviewer: ```suggestion
Throws a partial error (error code 206) if one or more document deletions fail. 
```
